utils/tasks.js

```diff
@@ -13,7 +13,7 @@ const fromFirestoreData = async (task) => {
     assignee = await getUsername(assignee)
   }
 
-  if (participants) {
+  if (Array.isArray(participants)) {
```

Comment:
Re: above

Reply:
Sorry for the trouble, The changes are not from the current PR

Reply:
@rashid301 , this is something you might find it useful